### PR TITLE
Combine `start` and `start:ext` to serve both at the same time

### PR DESCRIPTION
**Pull request by lukaw3d**
Closes https://github.com/oasisprotocol/oasis-wallet-web/issues/828

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,10 +8,9 @@
   },
   "scripts": {
     "clean": "rm -rf build/ .parcel-cache",
-    "start": "REACT_APP_META_CSP= parcel --port 3000 --target web --dist-dir build",
     "parcel:prod": "parcel build --target web --dist-dir build",
     "clean:ext": "rm -rf build-ext/ .parcel-cache",
-    "start:ext": "HMR_PORT=2222 EXTENSION_CSP=`yarn --silent print-extension-csp` parcel --host localhost --target ext --dist-dir build-ext",
+    "start": "REACT_APP_META_CSP= HMR_PORT=2222 EXTENSION_CSP=`yarn --silent print-extension-csp` parcel --host localhost --port 3000 --dist-dir build-dev",
```
Comment (ysz) [inline package.json:13]: How about we add `REACT_APP_BYPASS_LOCAL=1` such that `yarn start` works out of the box by default without local node?

**Review by ysz — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (lukaw3d) [reply package.json:13]: edit: REACT_APP_LOCALNET=

File: package.json [hunk 2]
```diff
@@ -8,18 +8,17 @@
...
     "build:ext": "yarn clean:ext && EXTENSION_CSP=`yarn --silent print-extension-csp` parcel build --target ext --dist-dir build-ext && node ./internals/scripts/validate-ext-manifest.js",
```
Comment (lukaw3d) [inline package.json:14]: Hm. Extension is broken if built with `yarn build:ext`, but it works from `yarn start`

Comment (lukaw3d) [reply package.json:14]: In background page: `Uncaught Error: Cannot find module 'fj44w'`
background.8e475e86.js contains: `"webext-redux": "fj44w",`
popup.9c7b8803.js contains:  `"webext-redux": "fj44w",`
popup.c6ac691b.js contains: `fj44w: [ function(q, H, ee) {`

Comment (lukaw3d) [reply package.json:14]: Upstream issue https://github.com/parcel-bundler/parcel/issues/8071#issuecomment-1148069991

Comment (lukaw3d) [reply package.json:14]: Deferring to https://github.com/oasisprotocol/oasis-wallet-web/issues/853

**Pull request merged**